Makefile

```diff
@@ -0,0 +1,29 @@
+REGISTRY?=registry.devshift.net
```

Comment:
Is this Makefile needed? It was probably copied from server repo based on image names.

Reply:
Removed